components/Editor/index.tsx

```diff
@@ -241,7 +243,11 @@ const Editor = ({ readOnly = false }: Props) => {
   return (
     <>
       <div className="bg-gray-100 dark:bg-black-700 rounded-lg">
-        <div className="flex flex-col md:flex-row">
+        <div
+          className={`flex flex-col md:flex-row ${
+            isFullScreen ? 'h-[92vh]' : ''
+          }`}
```

Comment:
can you use the `cn` object for assigning conditional classes (`import cn from 'classnames'`)